README.md

```diff
@@ -0,0 +1,86 @@
+# Team 5690 SubZero Robotics 2023 Command Robot
+
+## Table of contents
+
+- [Team 5690 SubZero Robotics 2023 Command Robot](#team-5690-subzero-robotics-2023-command-robot)
+  - [Table of contents](#table-of-contents)
+  - [About](#about)
+  - [Getting started](#getting-started)
+    - [Prerequisites](#prerequisites)
+  - [Making changes](#making-changes)
+    - [Cloning the repo](#cloning-the-repo)
+      - [Issues](#issues)
+      - [Branches](#branches)
+      - [Adding commits](#adding-commits)
+      - [Pushing commits](#pushing-commits)
+      - [Pulling branches](#pulling-branches)
+      - [Pull requests](#pull-requests)
+  - [Contact](#contact)
+
+## About
+
+:warning: **TODO: Add about section**
```

Comment:
Want to add an about section first?

Reply:
I feel like we don't know enough about our robot's mechanical side yet to have one.